user_docs/en/changes.t2t

```diff
@@ -25,6 +25,11 @@ What's New in NVDA
   - When pressing ``ctrl+tab``, focus properly moves to the new current tab title. (#14986, @ABuffEr)
   -
 - In Adobe Reader, NVDA no longer ignores alternative text set on formulas in PDFs. (#12715)
+- Changed several gestures for BrailleSense devices to ensure the gestures work on French keyboards. (#15306)

```

Comment:
For clarity: the issue is not with French keyboard but with French braille table.

```suggestion
- Changed several gestures for BrailleSense devices to avoid conflicts with characters of the French braille table. (#15306)
```